modules/emr_node/emr_node.post_update.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @file
+ * Entity Meta Relation node post update file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Fix possibly misconfigured target_bundles values in some fields.
+ */
+function emr_node_post_update_00001(&$sandbox) {
```

Comment:
Running the update, the following error was thrown:

<img width="898" alt="update" src="https://user-images.githubusercontent.com/22908988/96424080-7fe2d380-11fa-11eb-8786-ca8c9206e768.png">

Reply:
fixed